api/v1_notifications.go

```diff
@@ -36,27 +48,25 @@ user_created_at as (
   AND is_current
 )
 SELECT
-    n.type,
-    n.group_id as group_id,
-    json_agg(
-      json_build_object(
-        'type', type,
-        'specifier', specifier,
-        'timestamp', EXTRACT(EPOCH FROM timestamp),
-        'data', data
-      )
-      ORDER BY timestamp DESC
-    )::jsonb as actions,
-    CASE
-      WHEN user_seen.seen_at is not NULL THEN now()::timestamp != user_seen.seen_at
-      ELSE EXISTS(SELECT 1 from notification_seen ns where ns.user_id = @user_id)
-    END::boolean as is_seen,
-
-    CASE
-      WHEN user_seen.seen_at != now()::timestamp THEN EXTRACT(EPOCH FROM user_seen.seen_at)
-      ELSE null
-    END AS seen_at
-
+	n.type,
+	n.group_id as group_id,
+	json_agg(
+		json_build_object(
+			'type', type,
+			'specifier', specifier,
+			'timestamp', EXTRACT(EPOCH FROM timestamp),
+			'data', data
+		)
+		ORDER BY timestamp DESC
+	)::jsonb as actions,
+	CASE
+		WHEN user_seen.seen_at is not NULL THEN now()::timestamp != user_seen.seen_at
+		ELSE EXISTS(SELECT 1 from notification_seen ns where ns.user_id = @user_id)
```

Comment:
super nit picky (and i know predates this) but maybe keep casing consistent? (FROM, AS, WHERE, IS NOT)